src/test/java/resources/operationsBatch.feature

```diff
@@ -0,0 +1,138 @@
+@gov
+Feature: Operations APP related feature
+
+  @ops-batch-setup @ops-batch-teardown
+  Scenario: Batches API no filter test
+    Given I have tenant as "rhino"
+    When I call the operations-app auth endpoint with username: "mifos" and password: "password"
```

Comment:
This should be parameterized into application.yml so that secrets can be managed effectively.

Reply:
This?

Reply:
yes

Reply:
[PHEE-423](https://fynarfin.atlassian.net/browse/PHEE-423) Created backlog ticket